### ISSUE TITLE
2024-12-19, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#25]
# Today's Arxiv Papers

(1) [Thinking in Space: How Multimodal Large Language Models See, Remember, and Recall Spaces](http://arxiv.org/abs/2412.14171v1), 2024-12-18

(2) [MetaMorph: Multimodal Understanding and Generation via Instruction Tuning](http://arxiv.org/abs/2412.14164v1), 2024-12-18

(3) [TheAgentCompany: Benchmarking LLM Agents on Consequential Real World Tasks](http://arxiv.org/abs/2412.14161v1), 2024-12-18

(4) [Advanced Reasoning and Transformation Engine for Multi-Step Insight Synthesis in Data Analytics with Large Language Models](http://arxiv.org/abs/2412.14146v1), 2024-12-18

(5) [LLMs can realize combinatorial creativity: generating creative ideas via LLMs for scientific research](http://arxiv.org/abs/2412.14141v1), 2024-12-18

(6) [GLIDER: Grading LLM Interactions and Decisions using Explainable Ranking](http://arxiv.org/abs/2412.14140v1), 2024-12-18

(7) [Design choices made by LLM-based test generators prevent them from finding bugs](http://arxiv.org/abs/2412.14137v1), 2024-12-18

(8) [Scaling of Search and Learning: A Roadmap to Reproduce o1 from Reinforcement Learning Perspective](http://arxiv.org/abs/2412.14135v1), 2024-12-18

(9) [Alignment faking in large language models](http://arxiv.org/abs/2412.14093v1), 2024-12-18

(10) [Future Research Avenues for Artificial Intelligence in Digital Gaming: An Exploratory Report](http://arxiv.org/abs/2412.14085v1), 2024-12-18

(11) [Rango: Adaptive Retrieval-Augmented Proving for Automated Software Verification](http://arxiv.org/abs/2412.14063v1), 2024-12-18

(12) [Understanding and Evaluating Trust in Generative AI and Large Language Models for Spreadsheets](http://arxiv.org/abs/2412.14062v1), 2024-12-18

(13) [A Review of Multimodal Explainable Artificial Intelligence: Past, Present and Future](http://arxiv.org/abs/2412.14056v1), 2024-12-18

(14) [Digestion Algorithm in Hierarchical Symbolic Forests: A Fast Text Normalization Algorithm and Semantic Parsing Framework for Specific Scenarios and Lightweight Deployment](http://arxiv.org/abs/2412.14054v1), 2024-12-18

(15) [Cross-Lingual Transfer of Debiasing and Detoxification in Multilingual LLMs: An Extensive Investigation](http://arxiv.org/abs/2412.14050v1), 2024-12-18

(16) [CAD-Recode: Reverse Engineering CAD Code from Point Clouds](http://arxiv.org/abs/2412.14042v1), 2024-12-18

(17) [Hansel: Output Length Controlling Framework for Large Language Models](http://arxiv.org/abs/2412.14033v1), 2024-12-18

(18) [Discovering maximally consistent distribution of causal tournaments with Large Language Models](http://arxiv.org/abs/2412.14019v1), 2024-12-18

(19) [Towards an optimised evaluation of teachers' discourse: The case of engaging messages](http://arxiv.org/abs/2412.14011v1), 2024-12-18

(20) [Cognition Chain for Explainable Psychological Stress Detection on Social Media](http://arxiv.org/abs/2412.14009v1), 2024-12-18

(21) [FarExStance: Explainable Stance Detection for Farsi](http://arxiv.org/abs/2412.14008v1), 2024-12-18

(22) [InstructSeg: Unifying Instructed Visual Segmentation with Multi-modal Large Language Models](http://arxiv.org/abs/2412.14006v1), 2024-12-18

(23) [Few-shot Steerable Alignment: Adapting Rewards and LLM Policies with Neural Processes](http://arxiv.org/abs/2412.13998v1), 2024-12-18

(24) [RAG for Effective Supply Chain Security Questionnaire Automation](http://arxiv.org/abs/2412.13988v1), 2024-12-18

(25) [Prompting Strategies for Enabling Large Language Models to Infer Causation from Correlation](http://arxiv.org/abs/2412.13952v1), 2024-12-18

(26) [Cracking the Code of Hallucination in LVLMs with Vision-aware Head Divergence](http://arxiv.org/abs/2412.13949v1), 2024-12-18

(27) [Real Classification by Description: Extending CLIP's Limits of Part Attributes Recognition](http://arxiv.org/abs/2412.13947v1), 2024-12-18

(28) [A Rose by Any Other Name: LLM-Generated Explanations Are Good Proxies for Human Explanations to Collect Label Distributions on NLI](http://arxiv.org/abs/2412.13942v1), 2024-12-18

(29) [Language verY Rare for All](http://arxiv.org/abs/2412.13924v1), 2024-12-18

(30) [Pipeline Analysis for Developing Instruct LLMs in Low-Resource Languages: A Case Study on Basque](http://arxiv.org/abs/2412.13922v1), 2024-12-18

(31) [Crabs: Consuming Resrouce via Auto-generation for LLM-DoS Attack under Black-box Settings](http://arxiv.org/abs/2412.13879v1), 2024-12-18

(32) [LLaVA-UHD v2: an MLLM Integrating High-Resolution Feature Pyramid via Hierarchical Window Transformer](http://arxiv.org/abs/2412.13871v1), 2024-12-18

(33) [Energy-Based Preference Model Offers Better Offline Alignment than the Bradley-Terry Preference Model](http://arxiv.org/abs/2412.13862v1), 2024-12-18

(34) [Domain-adaptative Continual Learning for Low-resource Tasks: Evaluation on Nepali](http://arxiv.org/abs/2412.13860v1), 2024-12-18

(35) [Zero-Shot Prompting and Few-Shot Fine-Tuning: Revisiting Document Image Classification Using Large Language Models](http://dx.doi.org/10.1007/978-3-031-78495-8_10), 2024-12-18

(36) [Do Language Models Understand Time?](http://arxiv.org/abs/2412.13845v1), 2024-12-18

(37) [RACQUET: Unveiling the Dangers of Overlooked Referential Ambiguity in Visual LLMs](http://arxiv.org/abs/2412.13835v1), 2024-12-18

(38) [Maybe you are looking for CroQS: Cross-modal Query Suggestion for Text-to-Image Retrieval](http://arxiv.org/abs/2412.13834v1), 2024-12-18

(39) [Prompt Categories Cluster for Weakly Supervised Semantic Segmentation](http://arxiv.org/abs/2412.13823v1), 2024-12-18

(40) [Nullu: Mitigating Object Hallucinations in Large Vision-Language Models via HalluSpace Projection](http://arxiv.org/abs/2412.13817v1), 2024-12-18

(41) [CAD-Assistant: Tool-Augmented VLLMs as Generic CAD Task Solvers?](http://arxiv.org/abs/2412.13810v1), 2024-12-18

(42) [A Comprehensive Evaluation of Parameter-Efficient Fine-Tuning on Method-Level Code Smell Detection](http://arxiv.org/abs/2412.13801v1), 2024-12-18

(43) [Mix-LN: Unleashing the Power of Deeper Layers by Combining Pre-LN and Post-LN](http://arxiv.org/abs/2412.13795v1), 2024-12-18

(44) [Physics Reasoner: Knowledge-Augmented Reasoning for Solving Physics Problems with Large Language Models](http://arxiv.org/abs/2412.13791v1), 2024-12-18

(45) [Knowledge Editing with Dynamic Knowledge Graphs for Multi-hop Question Answering](http://arxiv.org/abs/2412.13782v1), 2024-12-18

(46) [Meta-Reflection: A Feedback-Free Reflection Learning Framework](http://arxiv.org/abs/2412.13781v1), 2024-12-18

(47) [Designing an LLM-Based Copilot for Manufacturing Equipment Selection](http://arxiv.org/abs/2412.13774v1), 2024-12-18

(48) [Semantic Convergence: Harmonizing Recommender Systems via Two-Stage Alignment and Behavioral Semantic Tokenization](http://arxiv.org/abs/2412.13771v1), 2024-12-18

(49) [LLM-SEM: A Sentiment-Based Student Engagement Metric Using LLMS for E-Learning Platforms](http://arxiv.org/abs/2412.13765v1), 2024-12-18

(50) [RAG-RewardBench: Benchmarking Reward Models in Retrieval Augmented Generation for Preference Alignment](http://arxiv.org/abs/2412.13746v1), 2024-12-18

(51) [Text2Relight: Creative Portrait Relighting with Text Guidance](http://arxiv.org/abs/2412.13734v1), 2024-12-18

(52) [Federated Learning and RAG Integration: A Scalable Approach for Medical Large Language Models](http://arxiv.org/abs/2412.13720v1), 2024-12-18

(53) [Mitigating Adversarial Attacks in LLMs through Defensive Suffix Generation](http://arxiv.org/abs/2412.13705v1), 2024-12-18

(54) [Typhoon 2: A Family of Open Text and Multimodal Thai Large Language Models](http://arxiv.org/abs/2412.13702v1), 2024-12-18

(55) [Towards Efficient and Explainable Hate Speech Detection via Model Distillation](http://arxiv.org/abs/2412.13698v1), 2024-12-18

(56) [A2H: A UI Converter from Android to HarmonyOS Platform](http://arxiv.org/abs/2412.13693v1), 2024-12-18

(57) [ChinaTravel: A Real-World Benchmark for Language Agents in Chinese Travel Planning](http://arxiv.org/abs/2412.13682v1), 2024-12-18

(58) [On Enhancing Root Cause Analysis with SQL Summaries for Failures in Database Workload Replays at SAP HANA](http://arxiv.org/abs/2412.13679v1), 2024-12-18

(59) [AntiLeak-Bench: Preventing Data Contamination by Automatically Constructing Benchmarks with Updated Real-World Knowledge](http://arxiv.org/abs/2412.13670v1), 2024-12-18

(60) [Exploring Multi-Modal Integration with Tool-Augmented LLM Agents for Precise Causal Discovery](http://arxiv.org/abs/2412.13667v1), 2024-12-18

(61) [Evaluation of LLM Vulnerabilities to Being Misused for Personalized Disinformation Generation](http://arxiv.org/abs/2412.13666v1), 2024-12-18

(62) [PsyDT: Using LLMs to Construct the Digital Twin of Psychological Counselor with Personalized Counseling Style for Psychological Counseling](http://arxiv.org/abs/2412.13660v1), 2024-12-18

(63) [RelationField: Relate Anything in Radiance Fields](http://arxiv.org/abs/2412.13652v1), 2024-12-18

(64) [SCOPE: Optimizing Key-Value Cache Compression in Long-context Generation](http://arxiv.org/abs/2412.13649v1), 2024-12-18

(65) [G-VEval: A Versatile Metric for Evaluating Image and Video Captions Using GPT-4o](http://arxiv.org/abs/2412.13647v1), 2024-12-18

(66) [On the Role of Model Prior in Real-World Inductive Reasoning](http://arxiv.org/abs/2412.13645v1), 2024-12-18

(67) [Mind Your Theory: Theory of Mind Goes Deeper Than Reasoning](http://arxiv.org/abs/2412.13631v1), 2024-12-18

(68) [LIFT: Improving Long Context Understanding Through Long Input Fine-Tuning](http://arxiv.org/abs/2412.13626v1), 2024-12-18

(69) [Are LLMs Good Literature Review Writers? Evaluating the Literature Review Writing Ability of Large Language Models](http://arxiv.org/abs/2412.13612v1), 2024-12-18

(70) [Beyond Outcomes: Transparent Assessment of LLM Reasoning in Games](http://arxiv.org/abs/2412.13602v1), 2024-12-18

(71) [EvoWiki: Evaluating LLMs on Evolving Knowledge](http://arxiv.org/abs/2412.13582v1), 2024-12-18

(72) [Socio-Culturally Aware Evaluation Framework for LLM-Based Content Moderation](http://arxiv.org/abs/2412.13578v1), 2024-12-18

(73) [Generating Long-form Story Using Dynamic Hierarchical Outlining with Memory-Enhancement](http://arxiv.org/abs/2412.13575v1), 2024-12-18

(74) [Large Language Model Federated Learning with Blockchain and Unlearning for Cross-Organizational Collaboration](http://arxiv.org/abs/2412.13551v1), 2024-12-18

(75) [Bridging the User-side Knowledge Gap in Knowledge-aware Recommendations with Large Language Models](http://arxiv.org/abs/2412.13544v1), 2024-12-18

(76) [MetaRuleGPT: Recursive Numerical Reasoning of Language Models Trained with Simple Rules](http://arxiv.org/abs/2412.13536v1), 2024-12-18

(77) [Information-Theoretic Generative Clustering of Documents](http://arxiv.org/abs/2412.13534v1), 2024-12-18

(78) [ScamGPT-J: Inside the Scammer's Mind, A Generative AI-Based Approach Toward Combating Messaging Scams](http://arxiv.org/abs/2412.13528v1), 2024-12-18

(79) [ROMAS: A Role-Based Multi-Agent System for Database monitoring and Planning](http://arxiv.org/abs/2412.13520v1), 2024-12-18

(80) [Efficient Language-instructed Skill Acquisition via Reward-Policy Co-Evolution](http://arxiv.org/abs/2412.13492v1), 2024-12-18

(81) [Refining Salience-Aware Sparse Fine-Tuning Strategies for Language Models](http://arxiv.org/abs/2412.13488v1), 2024-12-18

(82) [A Statistical and Multi-Perspective Revisiting of the Membership Inference Attack in Large Language Models](http://arxiv.org/abs/2412.13475v1), 2024-12-18

(83) [Gradual Vigilance and Interval Communication: Enhancing Value Alignment in Multi-Agent Debates](http://arxiv.org/abs/2412.13471v1), 2024-12-18

(84) [Transducer Tuning: Efficient Model Adaptation for Software Tasks Using Code Property Graphs](http://arxiv.org/abs/2412.13467v1), 2024-12-18

(85) [Lightweight Safety Classification Using Pruned Language Models](http://arxiv.org/abs/2412.13435v1), 2024-12-18

(86) [Large Language Model Enhanced Recommender Systems: Taxonomy, Trend, Application and Future](http://arxiv.org/abs/2412.13432v1), 2024-12-18

(87) [Safeguarding System Prompts for LLMs](http://arxiv.org/abs/2412.13426v1), 2024-12-18

(88) [Generating Diverse Hypotheses for Inductive Reasoning](http://arxiv.org/abs/2412.13422v1), 2024-12-18

(89) [BotSim: LLM-Powered Malicious Social Botnet Simulation](http://arxiv.org/abs/2412.13420v1), 2024-12-18

(90) [An Exploratory Study of ML Sketches and Visual Code Assistants](http://arxiv.org/abs/2412.13386v1), 2024-12-17

(91) [An Automated Explainable Educational Assessment System Built on LLMs](http://arxiv.org/abs/2412.13381v1), 2024-12-17

(92) [DateLogicQA: Benchmarking Temporal Biases in Large Language Models](http://arxiv.org/abs/2412.13377v1), 2024-12-17

(93) [Extending LLMs to New Languages: A Case Study of Llama and Persian Adaptation](http://arxiv.org/abs/2412.13375v1), 2024-12-17

(94) [Concept-ROT: Poisoning Concepts in Large Language Models with Model Editing](http://arxiv.org/abs/2412.13341v1), 2024-12-17

(95) [Unveiling the Secret Recipe: A Guide For Supervised Fine-Tuning Small LLMs](http://arxiv.org/abs/2412.13337v1), 2024-12-17

(96) [Experience of Training a 1.7B-Parameter LLaMa Model From Scratch](http://arxiv.org/abs/2412.13335v1), 2024-12-17

(97) [FastVLM: Efficient Vision Encoding for Vision Language Models](http://arxiv.org/abs/2412.13303v1), 2024-12-17

(98) [Hint Marginalization for Improved Reasoning in Large Language Models](http://arxiv.org/abs/2412.13292v1), 2024-12-17

(99) [Enhancing Persona Classification in Dialogue Systems: A Graph Neural Network Approach](http://arxiv.org/abs/2412.13283v1), 2024-12-17

(100) [JudgeBlender: Ensembling Judgments for Automatic Relevance Assessment](http://arxiv.org/abs/2412.13268v1), 2024-12-17
